plugins: Set platform IDs for non-USB/non-UDEV plugins

superm1 opened the pull request:
Recent changes in the daemon have caused these devices to start matching
as existing devices based on platform ID comparisons.

hughsie commented on plugins/amt/fu-plugin-amt.c:
```diff
@@ -416,7 +416,8 @@ fu_plugin_amt_create_device (GError **error)
 	dev = fu_device_new ();
-	fu_device_set_id (dev, "/dev/mei");
+	fu_device_set_id (dev, "/dev/mei0");
+	fu_device_set_platform_id (dev, "/dev");
```
I think you just want one call to `fu_device_set_platform_id (dev, "/dev/mei0");` -- if you don't have any GUIDs set then the value set as the platform ID is also just used as the ID.

superm1 replied:
fixed

hughsie commented on plugins/synapticsmst/fu-plugin-synapticsmst.c:
```diff
@@ -152,6 +152,7 @@ fu_plugin_synaptics_add_device (FuPlugin *plugin,
 	dev_id_str = g_strdup_printf ("MST-%s-%s-%u-%u",
 				      kind_str, aux_node, layer, rad);
 	fu_device_set_id (dev, dev_id_str);
+	fu_device_set_platform_id (dev, aux_node);
```
Is this really the hardware ID? It's designed to be something based on the physical connection.

superm1 replied:
Yeah the aux node will change if it's physically connected differently.

hughsie commented on plugins/uefi/fu-uefi-device.c:
```diff
@@ -420,6 +420,7 @@ fu_uefi_device_new_from_entry (const gchar *entry_path)
 	id = g_strdup_printf ("UEFI-%s-dev%" G_GUINT64_FORMAT,
 			      self->fw_class, self->fmp_hardware_instance);
 	fu_device_set_id (FU_DEVICE (self), id);
+	fu_device_set_platform_id (FU_DEVICE (self), "/sys/firmware/efi/esrt/");
```
Maybe just use `entry_path`? The platform ID should be unique for all devices too.

superm1 replied:
Yep, adjusted

hughsie commented:
> devices to start matching as existing devices

This sounds like a bug; if the platform ID is NULL we're never supposed to compare it... Got any more info?

superm1 commented:
Here is the failure:
```
18:40:24:0555 Fu  ignoring device 370e10407b1f04ade798a9f1d3e1fa57c67750c3 [uefi] existing device 088df415cdee883ec89563e41e6d495924250174 [amt] already exists
```

I bisected down:
```
git bisect start
# bad: [4631bd7f212118f0445c4ecfd5b5be7423f5317c] trivial: fu-device: Downgrade warnings about overwiting same name to debug
git bisect bad 4631bd7f212118f0445c4ecfd5b5be7423f5317c
# good: [4937ff5499850dc8b31b4e7e315826e7f1750f20] trivial: Add a self test to test the quirk database performance
git bisect good 4937ff5499850dc8b31b4e7e315826e7f1750f20
# good: [fd406616f762cbf2c1b272ae514b98522c5c0758] trivial: Create fu_device_list_replace() for future re-use
git bisect good fd406616f762cbf2c1b272ae514b98522c5c0758
# good: [effcc7a7a7a7d4b21b348c1ff4c04004b29ebeed] trivial: Fix regression when enumerating udev subsystems
git bisect good effcc7a7a7a7d4b21b348c1ff4c04004b29ebeed
# good: [1d894f13cc7cc5a9daf5a819b08bf38c7844b18b] trivial: Move the hwids debug functionality to fwupdtool
git bisect good 1d894f13cc7cc5a9daf5a819b08bf38c7844b18b
# good: [457888e4183aa7bb7e6d5e3b2d314aaede3f3c55] Always use the same HardwareIDs as Windows
git bisect good 457888e4183aa7bb7e6d5e3b2d314aaede3f3c55
# bad: [3a8d532855f7f44786250dd1b863fb8cfa60daa2] Allow the device list to take care of waiting for the device replug
git bisect bad 3a8d532855f7f44786250dd1b863fb8cfa60daa2
# first bad commit: [3a8d532855f7f44786250dd1b863fb8cfa60daa2] Allow the device list to take care of waiting for the device replug
```

Which brings it down to this matching of platform ID here being the cause:
https://github.com/hughsie/fwupd/commit/3a8d532855f7f44786250dd1b863fb8cfa60daa2#diff-289d6593b8402bcd4e79e7429a53562bR493


Full failure log of it:
```
18:40:12:0797 Fu  performing coldplug_prepare() on dell
18:40:13:0329 Dell Toggled dock mode to 1
18:40:13:0329 Fu  got coldplug delay of 2000ms, global maximum is now 2000ms
18:40:13:0329 Fu  sleeping for 2000ms
18:40:15:0329 As  run FuEngine:coldplug
18:40:15:0329 As  run FuEngine:coldplug{amt}
18:40:15:0330 Fu  performing coldplug() on amt
18:40:15:0342 Amt call write length = 12
18:40:15:0366 Amt write success
18:40:15:0367 Amt call read length = 5120
18:40:15:0367 Amt read succeeded with result 525
18:40:15:0367 Fu  using 088df415cdee883ec89563e41e6d495924250174 for /dev/mei
18:40:15:0367 Fu  using 230c8b18-8d9b-53ec-838b-6cfc0383493a for main-system-firmware
18:40:15:0367 Amt call write length = 12
18:40:15:0387 Amt write success
18:40:15:0387 Amt call read length = 5120
18:40:15:0387 Amt read succeeded with result 20
18:40:15:0388 Fu  emit added from amt: 088df415cdee883ec89563e41e6d495924250174
18:40:15:0388 Fu  performing fu_plugin_device_registered() on dell
18:40:15:0389 Fu  performing fu_plugin_device_registered() on uefi
18:40:15:0389 Fu  performing fu_plugin_device_registered() on thunderbolt_power
18:40:15:0389 Fu  ::added 088df415cdee883ec89563e41e6d495924250174
18:40:15:0389 As  run FuEngine:coldplug{udev}
18:40:15:0389 As  run FuEngine:coldplug{dell_esrt}
18:40:15:0389 As  run FuEngine:coldplug{ebitdo}
18:40:15:0389 As  run FuEngine:coldplug{steelseries}
18:40:15:0390 As  run FuEngine:coldplug{dfu}
18:40:15:0390 As  run FuEngine:coldplug{nitrokey}
18:40:15:0390 As  run FuEngine:coldplug{csr}
18:40:15:0390 As  run FuEngine:coldplug{unifying}
18:40:15:0390 Fu  performing coldplug() on unifying
18:40:15:0395 Unifying UDEV add /dev/hidraw0 = /sys/devices/pci0000:00/0000:00:15.1/i2c_designware.1/i2c-7/i2c-DELL07E6:00/0018:06CB:76AF.0001/hidraw/hidraw0
18:40:15:0396 Unifying not a matching vid: 06cb
18:40:15:0396 Unifying UDEV add /dev/hidraw4 = /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:02.0/0000:39:00.0/usb3/3-1/3-1.4/3-1.4:1.2/0003:046D:C52B.0028/0003:046D:1025.002A/hidraw/hidraw4
18:40:15:0397 Fu  using 77d843f7-682c-57e8-8e29-584f5b4f52a1 for USB\VID_046D&PID_C52B
18:40:15:0397 Fu  using 339bffd9a1e32203b88a9f0f28d140a9171d145a for /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:02.0/0000:39:00.0/usb3/3-1/3-1.4/3-1.4:1.2/0003:046D:C52B.0028/0003:046D:1025.002A/hidraw/hidraw4
18:40:15:0397 Fu  using 705f53c6-eaf7-53f2-936a-a9a8024f6c35 for UFY\VID_046D&PID_1025
18:40:15:0397 Unifying UFY\VID_046D&PID_1025 not supported, so ignoring device
18:40:15:0397 Unifying UDEV add /dev/hidraw3 = /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:02.0/0000:39:00.0/usb3/3-1/3-1.4/3-1.4:1.2/0003:046D:C52B.0028/0003:046D:4002.0029/hidraw/hidraw3
18:40:15:0398 Fu  using 77d843f7-682c-57e8-8e29-584f5b4f52a1 for USB\VID_046D&PID_C52B
18:40:15:0398 Fu  using 4b58d9d5ad88a38af54445a2b8b2d51964b7d018 for /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:02.0/0000:39:00.0/usb3/3-1/3-1.4/3-1.4:1.2/0003:046D:C52B.0028/0003:046D:4002.0029/hidraw/hidraw3
18:40:15:0398 Fu  using 1e9e99a8-56ed-5d95-865e-6d00e9f9bc9e for UFY\VID_046D&PID_4002
18:40:15:0399 Unifying UFY\VID_046D&PID_4002 not supported, so ignoring device
18:40:15:0399 Unifying UDEV add /dev/hidraw2 = /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:02.0/0000:39:00.0/usb3/3-1/3-1.4/3-1.4:1.2/0003:046D:C52B.0028/hidraw/hidraw2
18:40:15:0399 Fu  using 3c0149ca772a58689863de0f73c1568891eb8347 for /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:02.0/0000:39:00.0/usb3/3-1/3-1.4
18:40:15:0400 Unifying device /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:02.0/0000:39:00.0/usb3/3-1/3-1.4 added
18:40:15:0400 Unifying opening unifying device using /dev/hidraw2
18:40:15:0400 Fu  using 77d843f7-682c-57e8-8e29-584f5b4f52a1 for USB\VID_046D&PID_C52B
18:40:15:0400 Fu  using 9d131a0c-a606-580f-8eda-80587250b8d6 for counterpart USB\VID_046D&PID_AAAA
18:40:15:0408 Unifying Unifying Receiver
  DeviceId:             3c0149ca772a58689863de0f73c1568891eb8347
  Guid:                 77d843f7-682c-57e8-8e29-584f5b4f52a1
  Guid:                 9d131a0c-a606-580f-8eda-80587250b8d6
  Summary:              A miniaturised USB wireless receiver
  Plugin:               unifying
  Flags:                updatable
  Vendor:               Logitech
  VendorId:             USB:0x046D
  Version:              RQR12.07_B0029
  VersionBootloader:    BOT01.02_B0014
  Icon:                 preferences-desktop-keyboard
  platform-id:          /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:02.0/0000:39:00.0/usb3/3-1/3-1.4
  Type:			runtime
  Flags:		active,is-open,requires-detach,detach-will-replug
  HidppVersion:		1.00
  HidppId:		0xff
  UdevDevice:		18

18:40:15:0408 As  run FuPluginLu:added{/sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:02.0/0000:39:00.0/usb3/3-1/3-1.4}
18:40:15:0408 Fu  emit added from unifying: 3c0149ca772a58689863de0f73c1568891eb8347
18:40:15:0408 Fu  performing fu_plugin_device_registered() on dell
18:40:15:0408 Fu  performing fu_plugin_device_registered() on uefi
18:40:15:0408 Fu  performing fu_plugin_device_registered() on thunderbolt_power
18:40:15:0408 Fu  ::added 3c0149ca772a58689863de0f73c1568891eb8347
18:40:15:0408 Unifying UDEV add /dev/hidraw5 = /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:02.0/0000:39:00.0/usb3/3-1/3-1.7/3-1.7:1.0/0003:413C:3016.002B/hidraw/hidraw5
18:40:15:0409 Unifying not a matching vid: 013c
18:40:15:0409 As  run FuEngine:coldplug{superio}
18:40:15:0409 Fu  performing coldplug() on superio
18:40:15:0409 As  run FuEngine:coldplug{wacomhid}
18:40:15:0409 As  run FuEngine:coldplug{altos}
18:40:15:0409 As  run FuEngine:coldplug{upower}
18:40:15:0409 As  run FuEngine:coldplug{colorhug}
18:40:15:0409 As  run FuEngine:coldplug{thunderbolt}
18:40:15:0409 Fu  performing coldplug() on thunderbolt
18:40:15:0410 As  run FuEngine:coldplug{dell}
18:40:15:0410 Fu  performing coldplug() on dell
18:40:15:0420 Dell TPM HW version: 0x0
18:40:15:0420 Dell TPM Status: 0x207
18:40:15:0420 Dell Creating primary TPM GUID 07e6-2.0 and secondary TPM GUID 07e6-1.2
18:40:15:0420 Fu  using 370e10407b1f04ade798a9f1d3e1fa57c67750c3 for DELL-cb8da68d-cd80-5f5b-8fef-038383adbb83lu
18:40:15:0420 Dell XPS 13 9370 TPM 2.0 updating disabled due to TPM ownership
18:40:15:0420 Fu  emit device-register from dell: 370e10407b1f04ade798a9f1d3e1fa57c67750c3
18:40:15:0421 Fu  performing fu_plugin_device_registered() on dell
18:40:15:0421 Fu  performing fu_plugin_device_registered() on uefi
18:40:15:0421 Fu  emit added from uefi: 370e10407b1f04ade798a9f1d3e1fa57c67750c3
18:40:15:0421 Fu  performing fu_plugin_device_registered() on dell
18:40:15:0421 Fu  performing fu_plugin_device_registered() on thunderbolt_power
18:40:15:0421 Fu  ignoring device 370e10407b1f04ade798a9f1d3e1fa57c67750c3 [uefi] existing device 088df415cdee883ec89563e41e6d495924250174 [amt] already exists
18:40:15:0421 Fu  performing fu_plugin_device_registered() on thunderbolt_power
18:40:15:0421 Dell System 07e6 does not offer TPM modeswitching
18:40:15:0421 As  run FuEngine:coldplug{flashrom}
18:40:15:0421 As  run FuEngine:coldplug{redfish}
18:40:15:0421 As  run FuEngine:coldplug{nvme}
18:40:15:0421 As  run FuEngine:coldplug{uefi}
18:40:15:0421 Fu  performing coldplug() on uefi
18:40:15:0426 Fu  using 8a21cacfb0a8d2b30c5ee9290eb71db021619f8b for UEFI-7ceaf7a8-0611-4480-9e30-64d8de420c7c-dev0
18:40:15:0426 Fu  using 230c8b18-8d9b-53ec-838b-6cfc0383493a for main-system-firmware
18:40:15:0426 Fu  emit added from uefi: 8a21cacfb0a8d2b30c5ee9290eb71db021619f8b
18:40:15:0426 Fu  setting parent of Intel AMT [unprovisioned] [088df415cdee883ec89563e41e6d495924250174] to be XPS 13 9370 System Firmware [8a21cacfb0a8d2b30c5ee9290eb71db021619f8b]
18:40:15:0427 Fu  performing fu_plugin_device_registered() on dell
18:40:15:0427 Fu  performing fu_plugin_device_registered() on thunderbolt_power
18:40:15:0427 Fu  ignoring device 8a21cacfb0a8d2b30c5ee9290eb71db021619f8b [uefi] existing device 088df415cdee883ec89563e41e6d495924250174 [amt] already exists
18:40:15:0496 Uefi ESP mountpoint set as /boot/efi
18:40:15:0498 Uefi SecureBoot is: Disabled
18:40:15:0498 Uefi UX Capsule support : Enabled
18:40:15:0498 As  run FuEngine:coldplug{thunderbolt_power}
18:40:15:0498 Fu  performing coldplug() on thunderbolt_power
18:40:15:0502 ThunderboltPower Bolt force power support: 1
18:40:15:0507 ThunderboltPower Bolt force power support: 1
18:40:15:0508 ThunderboltPower Setting force power to 1 using bolt
18:40:15:0533 ThunderboltPower Setting timeout to 20 seconds
18:40:15:0534 As  run FuEngine:coldplug{synapticsmst}
18:40:15:0535 Fu  performing coldplug() on synapticsmst
18:40:15:0539 SynapticsMST Adding direct device MST-DIRECT-drm_dp_aux1-0-0
18:40:15:0561 SynapticsMST BoardID 112
18:40:15:0566 Fu  using 7b3cbc344094ace839263dd4711c84ea99bb6dc3 for MST-DIRECT-drm_dp_aux1-0-0
18:40:15:0567 Fu  using 78c73924-821a-5e6f-af82-86df6f67b182 for MST-wd15-vmm3332-274
18:40:15:0567 Fu  using 02b77ccb-03ab-521f-b98e-ef05395e9fa2 for MST-tb16-vmm3332-274
18:40:15:0567 Fu  using a6f00ca7-54b7-5d51-9d6e-543d8df85376 for MST-tb18-vmm3332-274
18:40:15:0567 Fu  emit added from synapticsmst: 7b3cbc344094ace839263dd4711c84ea99bb6dc3
18:40:15:0567 Fu  performing fu_plugin_device_registered() on dell
18:40:15:0567 Fu  performing fu_plugin_device_registered() on uefi
18:40:15:0567 Fu  performing fu_plugin_device_registered() on thunderbolt_power
18:40:15:0567 Fu  ignoring device 7b3cbc344094ace839263dd4711c84ea99bb6dc3 [synapticsmst] existing device 088df415cdee883ec89563e41e6d495924250174 [amt] already exists
18:40:15:0598 SynapticsMST No cascade device found: remote command failed: 3
18:40:15:0621 SynapticsMST No cascade device found: remote command failed: 3
18:40:16:0007 SynapticsMST No device found on drm_dp_aux2: failed to read device: failed to read
18:40:16:0008 SynapticsMST No device found on drm_dp_aux0: no device
18:40:16:0008 Fu  performing coldplug_cleanup() on dell
18:40:16:0524 Dell Toggled dock mode to 0
18:40:16:0524 Fu  using plugins: amt, udev, ebitdo, steelseries, dfu, nitrokey, csr, unifying, superio, wacomhid, altos, upower, colorhug, thunderbolt, dell, flashrom, nvme, uefi, thunderbolt_power, synapticsmst
18:40:17:0495 Dell Dock description: IE_Dock
18:40:17:0495 Dell Dock flash pkg ver: 0x43
18:40:17:0495 Dell Dock cable type: 1
18:40:17:0495 Dell Dock location: 2
18:40:17:0495 Dell Dock component count: 3
18:40:17:0495 Dell Dock component 0: Dock1,EC,MIPS32,IE_Dock,0 :Query 2 0 2 2 0 (version 0xffffffff)
18:40:17:0495 Dell Dock component 1: Dock1,Cable,Cyp,IE_Cable,0 :Query 2 2 2 1 0 (version 0xffffffff)
18:40:17:0495 Dell Dock component 2: Dock1,PC,TI,IE_Dock,0 :Query 2 1 0 2 0 (version 0x10108)
18:40:17:0495 Fu  using 3ec64779e294dc058f2393efaa5c8db713aaaf67 for DELL-8ba2b709-6f97-47fc-b7e7-6a87b578fe25lu
18:40:17:0495 Fu  emit device-register from dell: 3ec64779e294dc058f2393efaa5c8db713aaaf67
18:40:17:0495 Fu  performing fu_plugin_device_registered() on dell
18:40:17:0495 Fu  performing fu_plugin_device_registered() on uefi
18:40:17:0495 Fu  emit added from uefi: 3ec64779e294dc058f2393efaa5c8db713aaaf67
18:40:17:0495 Fu  performing fu_plugin_device_registered() on dell
18:40:17:0495 Fu  performing fu_plugin_device_registered() on thunderbolt_power
18:40:17:0495 Fu  ignoring device 3ec64779e294dc058f2393efaa5c8db713aaaf67 [uefi] existing device 088df415cdee883ec89563e41e6d495924250174 [amt] already exists
18:40:17:0495 Fu  performing fu_plugin_device_registered() on thunderbolt_power
18:40:17:0495 Fu  using 8e5d18ee7aa0401befa8a18c4d84c573ee249e22 for DELL-e7ca1f36-bf73-4574-afe6-a4ccacabf479lu
18:40:17:0495 Fu  emit device-register from dell: 8e5d18ee7aa0401befa8a18c4d84c573ee249e22
18:40:17:0495 Fu  performing fu_plugin_device_registered() on dell
18:40:17:0495 Fu  performing fu_plugin_device_registered() on uefi
18:40:17:0495 Fu  emit added from uefi: 8e5d18ee7aa0401befa8a18c4d84c573ee249e22
18:40:17:0495 Fu  performing fu_plugin_device_registered() on dell
18:40:17:0495 Fu  performing fu_plugin_device_registered() on thunderbolt_power
18:40:17:0495 Fu  ignoring device 8e5d18ee7aa0401befa8a18c4d84c573ee249e22 [uefi] existing device 088df415cdee883ec89563e41e6d495924250174 [amt] already exists
18:40:17:0495 Fu  performing fu_plugin_device_registered() on thunderbolt_power
18:40:17:0495 Fu  scheduling a recoldplug
18:40:17:0495 Fu  using 54c916aa848583256a9b6959446d0890818cbe18 for usb:00:01:02
18:40:17:0495 Fu  using e48b8fad-2d2d-55e5-a0f2-1b431b35f3fd for USB\VID_0BDA&PID_8153&REV_3001
18:40:17:0495 Fu  using f62863ee-133d-511b-903c-d6dfa8f9fdf4 for USB\VID_0BDA&PID_8153
18:40:17:0495 Fu  using fb69fd72-4832-567d-9337-790ed4f68416 for USB\VID_0BDA
18:40:17:0495 Fu  no plugin specified for USB device 0bda:8153
18:40:17:0495 Fu  performing usb_device_added() on dfu
18:40:17:0495 Fu  using 54c916aa848583256a9b6959446d0890818cbe18 for usb:00:01:02
18:40:17:0495 Fu  using e48b8fad-2d2d-55e5-a0f2-1b431b35f3fd for USB\VID_0BDA&PID_8153&REV_3001
18:40:17:0495 Fu  using f62863ee-133d-511b-903c-d6dfa8f9fdf4 for USB\VID_0BDA&PID_8153
18:40:17:0495 Fu  using fb69fd72-4832-567d-9337-790ed4f68416 for USB\VID_0BDA
18:40:17:0496 Fu  dfu ignoring: 0bda:8153 is not supported: no DFU interfaces
18:40:17:0496 Fu  using 672c087de09848d9e7ee32aa1dea2fbeb8b81e6b for usb:00:01
18:40:17:0496 Fu  using 6b10e07d-2701-5069-bc31-c3e4ad7c595f for USB\VID_0424&PID_5537&REV_6080
18:40:17:0496 Fu  using 11c69e37-e465-5478-a8e4-2343146e4bb9 for USB\VID_0424&PID_5537
18:40:17:0496 Fu  using 738f310c-53fc-549d-844a-b3297139cad5 for USB\VID_0424
18:40:17:0496 Fu  no plugin specified for USB device 0424:5537
18:40:17:0496 Fu  performing usb_device_added() on dfu
18:40:17:0496 Fu  using 672c087de09848d9e7ee32aa1dea2fbeb8b81e6b for usb:00:01
18:40:17:0496 Fu  using 6b10e07d-2701-5069-bc31-c3e4ad7c595f for USB\VID_0424&PID_5537&REV_6080
18:40:17:0496 Fu  using 11c69e37-e465-5478-a8e4-2343146e4bb9 for USB\VID_0424&PID_5537
18:40:17:0496 Fu  using 738f310c-53fc-549d-844a-b3297139cad5 for USB\VID_0424
18:40:17:0496 Fu  dfu ignoring: 0424:5537 is not supported: no DFU interfaces
18:40:17:0496 Fu  using b74efe76cee83c70a7009680bc448b2d396c2ba1 for usb:00
18:40:17:0496 Fu  using bdc6f3e7-7259-54c9-9aec-4982c31734f7 for USB\VID_1D6B&PID_0003&REV_0418
18:40:17:0496 Fu  using 69c66597-c59d-5593-8d85-b261deae893a for USB\VID_1D6B&PID_0003
18:40:17:0496 Fu  using 32343899-27e0-55e5-8f9f-12ff7541e488 for USB\VID_1D6B
18:40:17:0496 Fu  no plugin specified for USB device 1d6b:0003
18:40:17:0496 Fu  performing usb_device_added() on dfu
18:40:17:0496 Fu  using b74efe76cee83c70a7009680bc448b2d396c2ba1 for usb:00
18:40:17:0496 Fu  using bdc6f3e7-7259-54c9-9aec-4982c31734f7 for USB\VID_1D6B&PID_0003&REV_0418
18:40:17:0496 Fu  using 69c66597-c59d-5593-8d85-b261deae893a for USB\VID_1D6B&PID_0003
18:40:17:0496 Fu  using 32343899-27e0-55e5-8f9f-12ff7541e488 for USB\VID_1D6B
18:40:17:0496 Fu  dfu ignoring: 1d6b:0003 is not supported: no DFU interfaces
18:40:17:0496 Fu  using daa833f08c8b785b5f46be7143f81c8622987f1c for usb:00:01:07
18:40:17:0496 Fu  using dea15c0d-cc11-5185-ac3c-001e9ef4da0e for USB\VID_413C&PID_3016&REV_0007
18:40:17:0496 Fu  using 18be134c-ef8d-57af-9091-2e33ed49b9a8 for USB\VID_413C&PID_3016
18:40:17:0496 Fu  using 795e079d-093b-5503-aa59-35b832480e95 for USB\VID_413C
18:40:17:0496 Fu  no plugin specified for USB device 413c:3016
18:40:17:0496 Fu  performing usb_device_added() on dfu
18:40:17:0496 Fu  using daa833f08c8b785b5f46be7143f81c8622987f1c for usb:00:01:07
18:40:17:0496 Fu  using dea15c0d-cc11-5185-ac3c-001e9ef4da0e for USB\VID_413C&PID_3016&REV_0007
18:40:17:0496 Fu  using 18be134c-ef8d-57af-9091-2e33ed49b9a8 for USB\VID_413C&PID_3016
18:40:17:0496 Fu  using 795e079d-093b-5503-aa59-35b832480e95 for USB\VID_413C
18:40:17:0496 Fu  dfu ignoring: 413c:3016 is not supported: no DFU interfaces
18:40:17:0496 Fu  using c3857357b6f271e9254918e2d524d5a5b6f6b747 for usb:00:01:06
18:40:17:0496 Fu  using fca7e877-1835-5288-adf4-797733cfb167 for USB\VID_413C&PID_9009&REV_0100
18:40:17:0496 Fu  using c65c7184-4b64-5a73-9f1b-c564866d98d6 for USB\VID_413C&PID_9009
18:40:17:0496 Fu  using 795e079d-093b-5503-aa59-35b832480e95 for USB\VID_413C
18:40:17:0496 Fu  no plugin specified for USB device 413c:9009
18:40:17:0496 Fu  performing usb_device_added() on dfu
18:40:17:0496 Fu  using c3857357b6f271e9254918e2d524d5a5b6f6b747 for usb:00:01:06
18:40:17:0496 Fu  using fca7e877-1835-5288-adf4-797733cfb167 for USB\VID_413C&PID_9009&REV_0100
18:40:17:0497 Fu  using c65c7184-4b64-5a73-9f1b-c564866d98d6 for USB\VID_413C&PID_9009
18:40:17:0497 Fu  using 795e079d-093b-5503-aa59-35b832480e95 for USB\VID_413C
18:40:17:0497 Fu  dfu ignoring: 413c:9009 is not supported: no DFU interfaces
18:40:17:0497 Fu  using 970e85c8400ed95ec5c02b86e52a0ca4ce6fc051 for usb:00:01:05
18:40:17:0497 Fu  using 649d79d2-6579-5bf4-9327-6c1f2e13f2a0 for USB\VID_0BDA&PID_4014&REV_0001
18:40:17:0497 Fu  using de6f318b-5c72-5c06-b494-ebcdea9be4c3 for USB\VID_0BDA&PID_4014
18:40:17:0497 Fu  using fb69fd72-4832-567d-9337-790ed4f68416 for USB\VID_0BDA
18:40:17:0497 Fu  no plugin specified for USB device 0bda:4014
18:40:17:0497 Fu  performing usb_device_added() on dfu
18:40:17:0497 Fu  using 970e85c8400ed95ec5c02b86e52a0ca4ce6fc051 for usb:00:01:05
18:40:17:0497 Fu  using 649d79d2-6579-5bf4-9327-6c1f2e13f2a0 for USB\VID_0BDA&PID_4014&REV_0001
18:40:17:0497 Fu  using de6f318b-5c72-5c06-b494-ebcdea9be4c3 for USB\VID_0BDA&PID_4014
18:40:17:0497 Fu  using fb69fd72-4832-567d-9337-790ed4f68416 for USB\VID_0BDA
18:40:17:0497 Fu  dfu ignoring: 0bda:4014 is not supported: no DFU interfaces
18:40:17:0497 Unifying USB add usb:00:01:04
18:40:17:0497 Fu  using de63d46dd16f065cc712a6f6f1d9698854b15053 for usb:00:01:04
18:40:17:0497 Fu  using 39a2d8d5-9394-5e6a-b7fc-6fcebfe777aa for USB\VID_046D&PID_C52B&REV_1207
18:40:17:0497 Fu  using 77d843f7-682c-57e8-8e29-584f5b4f52a1 for USB\VID_046D&PID_C52B
18:40:17:0497 Fu  using 941f4133-39e5-56a0-9df6-902547436c5c for USB\VID_046D
18:40:17:0497 Fu  using 672c087de09848d9e7ee32aa1dea2fbeb8b81e6b for usb:00:01
18:40:17:0497 Fu  using fb64ee81-209b-57c0-bd54-cacbd5b4dc1e for USB\VID_0424&PID_2137&REV_6080
18:40:17:0497 Fu  using ebc240dd-eab3-5d0a-865c-56d4020c2532 for USB\VID_0424&PID_2137
18:40:17:0497 Fu  using 738f310c-53fc-549d-844a-b3297139cad5 for USB\VID_0424
18:40:17:0497 Fu  no plugin specified for USB device 0424:2137
18:40:17:0497 Fu  performing usb_device_added() on dfu
18:40:17:0497 Fu  using 672c087de09848d9e7ee32aa1dea2fbeb8b81e6b for usb:00:01
18:40:17:0497 Fu  using fb64ee81-209b-57c0-bd54-cacbd5b4dc1e for USB\VID_0424&PID_2137&REV_6080
18:40:17:0497 Fu  using ebc240dd-eab3-5d0a-865c-56d4020c2532 for USB\VID_0424&PID_2137
18:40:17:0497 Fu  using 738f310c-53fc-549d-844a-b3297139cad5 for USB\VID_0424
18:40:17:0497 Fu  dfu ignoring: 0424:2137 is not supported: no DFU interfaces
18:40:17:0497 Fu  using b74efe76cee83c70a7009680bc448b2d396c2ba1 for usb:00
18:40:17:0497 Fu  using 2a9862a2-7528-573b-9dd0-717861bc5450 for USB\VID_1D6B&PID_0002&REV_0418
18:40:17:0497 Fu  using 04056ce4-bb53-53d9-8089-3621cfa73e31 for USB\VID_1D6B&PID_0002
18:40:17:0497 Fu  using 32343899-27e0-55e5-8f9f-12ff7541e488 for USB\VID_1D6B
18:40:17:0497 Fu  no plugin specified for USB device 1d6b:0002
18:40:17:0497 Fu  performing usb_device_added() on dfu
18:40:17:0497 Fu  using b74efe76cee83c70a7009680bc448b2d396c2ba1 for usb:00
18:40:17:0497 Fu  using 2a9862a2-7528-573b-9dd0-717861bc5450 for USB\VID_1D6B&PID_0002&REV_0418
18:40:17:0497 Fu  using 04056ce4-bb53-53d9-8089-3621cfa73e31 for USB\VID_1D6B&PID_0002
18:40:17:0497 Fu  using 32343899-27e0-55e5-8f9f-12ff7541e488 for USB\VID_1D6B
18:40:17:0498 Fu  dfu ignoring: 1d6b:0002 is not supported: no DFU interfaces
18:40:17:0498 Fu  using b74efe76cee83c70a7009680bc448b2d396c2ba1 for usb:00
18:40:17:0498 Fu  using bdc6f3e7-7259-54c9-9aec-4982c31734f7 for USB\VID_1D6B&PID_0003&REV_0418
18:40:17:0498 Fu  using 69c66597-c59d-5593-8d85-b261deae893a for USB\VID_1D6B&PID_0003
18:40:17:0498 Fu  using 32343899-27e0-55e5-8f9f-12ff7541e488 for USB\VID_1D6B
18:40:17:0499 Fu  no plugin specified for USB device 1d6b:0003
18:40:17:0499 Fu  performing usb_device_added() on dfu
18:40:17:0499 Fu  using b74efe76cee83c70a7009680bc448b2d396c2ba1 for usb:00
18:40:17:0499 Fu  using bdc6f3e7-7259-54c9-9aec-4982c31734f7 for USB\VID_1D6B&PID_0003&REV_0418
18:40:17:0499 Fu  using 69c66597-c59d-5593-8d85-b261deae893a for USB\VID_1D6B&PID_0003
18:40:17:0499 Fu  using 32343899-27e0-55e5-8f9f-12ff7541e488 for USB\VID_1D6B
18:40:17:0500 Fu  dfu ignoring: 1d6b:0003 is not supported: no DFU interfaces
18:40:17:0500 Fu  using 0f62b6984e1559ed77613bf161e4069e74714baf for usb:00:07
18:40:17:0500 Fu  using a885db9f-f691-5141-94cf-67cee3d54483 for USB\VID_0489&PID_E0A2&REV_0001
18:40:17:0500 Fu  using 4e26b71c-c559-5255-af11-29bf03d2881b for USB\VID_0489&PID_E0A2
18:40:17:0500 Fu  using 8db3d159-4212-5077-aa94-c700c0da2b34 for USB\VID_0489
18:40:17:0500 Fu  no plugin specified for USB device 0489:e0a2
18:40:17:0500 Fu  performing usb_device_added() on dfu
18:40:17:0500 Fu  using 0f62b6984e1559ed77613bf161e4069e74714baf for usb:00:07
18:40:17:0500 Fu  using a885db9f-f691-5141-94cf-67cee3d54483 for USB\VID_0489&PID_E0A2&REV_0001
18:40:17:0500 Fu  using 4e26b71c-c559-5255-af11-29bf03d2881b for USB\VID_0489&PID_E0A2
18:40:17:0500 Fu  using 8db3d159-4212-5077-aa94-c700c0da2b34 for USB\VID_0489
18:40:17:0500 Fu  dfu ignoring: 0489:e0a2 is not supported: no DFU interfaces
18:40:17:0500 Fu  using ef5791f75bea792344af5e07f52d4bb54563221c for usb:00:05
18:40:17:0500 Fu  using 8786ee7a-f54d-50ec-b248-2dc213ae831c for USB\VID_0BDA&PID_58F4&REV_7605
18:40:17:0500 Fu  using 43b935bc-29e5-5d37-ba3f-932898a35276 for USB\VID_0BDA&PID_58F4
18:40:17:0500 Fu  using fb69fd72-4832-567d-9337-790ed4f68416 for USB\VID_0BDA
18:40:17:0500 Fu  no plugin specified for USB device 0bda:58f4
18:40:17:0500 Fu  performing usb_device_added() on dfu
18:40:17:0500 Fu  using ef5791f75bea792344af5e07f52d4bb54563221c for usb:00:05
18:40:17:0500 Fu  using 8786ee7a-f54d-50ec-b248-2dc213ae831c for USB\VID_0BDA&PID_58F4&REV_7605
18:40:17:0500 Fu  using 43b935bc-29e5-5d37-ba3f-932898a35276 for USB\VID_0BDA&PID_58F4
18:40:17:0500 Fu  using fb69fd72-4832-567d-9337-790ed4f68416 for USB\VID_0BDA
18:40:17:0500 Fu  dfu ignoring: 0bda:58f4 is not supported: no DFU interfaces
18:40:17:0500 Fu  using b74efe76cee83c70a7009680bc448b2d396c2ba1 for usb:00
18:40:17:0500 Fu  using 2a9862a2-7528-573b-9dd0-717861bc5450 for USB\VID_1D6B&PID_0002&REV_0418
18:40:17:0500 Fu  using 04056ce4-bb53-53d9-8089-3621cfa73e31 for USB\VID_1D6B&PID_0002
18:40:17:0500 Fu  using 32343899-27e0-55e5-8f9f-12ff7541e488 for USB\VID_1D6B
18:40:17:0500 Fu  no plugin specified for USB device 1d6b:0002
18:40:17:0501 Fu  performing usb_device_added() on dfu
18:40:17:0501 Fu  using b74efe76cee83c70a7009680bc448b2d396c2ba1 for usb:00
18:40:17:0501 Fu  using 2a9862a2-7528-573b-9dd0-717861bc5450 for USB\VID_1D6B&PID_0002&REV_0418
18:40:17:0501 Fu  using 04056ce4-bb53-53d9-8089-3621cfa73e31 for USB\VID_1D6B&PID_0002
18:40:17:0501 Fu  using 32343899-27e0-55e5-8f9f-12ff7541e488 for USB\VID_1D6B
18:40:17:0501 Fu  dfu ignoring: 1d6b:0002 is not supported: no DFU interfaces
18:40:17:0503 Fu  25 devices with subsystem pci
18:40:17:0503 Fu  using bf9718cf4fa19f3961b7a50b4d07e8b52b875ffa for /sys/devices/pci0000:00/0000:00:00.0
18:40:17:0504 Fu  using 578054c1-28e3-5524-88d5-3d7e75b9e505 for PCI\VEN_8086&DEV_5904&REV_02
18:40:17:0504 Fu  using e36100be-dc68-51fd-aeff-7a346103fac2 for PCI\VEN_8086&DEV_5904
18:40:17:0504 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0504 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:00.0
18:40:17:0504 Fu  performing udev_device_added() on udev
18:40:17:0504 Fu  performing udev_device_added() on nvme
18:40:17:0504 Fu  using ca9f93cb97772027f45ba73c86cf4980fb02745b for /sys/devices/pci0000:00/0000:00:02.0
18:40:17:0504 Fu  using 2886e312-afd8-5510-8993-12d568a85e00 for PCI\VEN_8086&DEV_5916&REV_02
18:40:17:0504 Fu  using 38f6c009-a25c-5b50-b3b5-fef4f9c6b846 for PCI\VEN_8086&DEV_5916
18:40:17:0504 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0504 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:02.0
18:40:17:0504 Fu  performing udev_device_added() on udev
18:40:17:0504 As  run FuPluginUdev:client-add{0x8086:0x5916}
18:40:17:0504 Udev adding udev device: /sys/devices/pci0000:00/0000:00:02.0
18:40:17:0504 Fu  using ca9f93cb97772027f45ba73c86cf4980fb02745b for /sys/devices/pci0000:00/0000:00:02.0
18:40:17:0504 Fu  using 3ec3df3a-2290-56e5-9d2f-eda62e9ab50b for 0x8086:0x5916
18:40:17:0504 Fu  waiting a small time for other plugins
18:40:17:0504 Fu  performing udev_device_added() on nvme
18:40:17:0504 Fu  using 112a8b739c07f881f8a89817cb307bd0e3dccb5a for /sys/devices/pci0000:00/0000:00:04.0
18:40:17:0504 Fu  using 85a59b0b-a7f6-5a3c-ae3e-a75c7c4be70b for PCI\VEN_8086&DEV_1903&REV_02
18:40:17:0504 Fu  using e13786e6-33f2-50d5-ac26-37b1aedc22e1 for PCI\VEN_8086&DEV_1903
18:40:17:0504 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0504 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:04.0
18:40:17:0504 Fu  performing udev_device_added() on udev
18:40:17:0504 Fu  performing udev_device_added() on nvme
18:40:17:0504 Fu  using 3884d6468cd29f78b66cd05768fa9f761761dcbf for /sys/devices/pci0000:00/0000:00:14.0
18:40:17:0504 Fu  using cc90ecbf-5f72-5259-b0d8-3f543e7fe566 for PCI\VEN_8086&DEV_9D2F&REV_21
18:40:17:0504 Fu  using a84a1a0f-f873-57dd-b064-e15858a66c9d for PCI\VEN_8086&DEV_9D2F
18:40:17:0505 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0505 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:14.0
18:40:17:0505 Fu  performing udev_device_added() on udev
18:40:17:0505 Fu  performing udev_device_added() on nvme
18:40:17:0505 Fu  using 6f15f1af4f441658c2c342406d55603414fa5aa2 for /sys/devices/pci0000:00/0000:00:14.2
18:40:17:0505 Fu  using e0ba9791-b861-5a05-a5e9-c393f04a683f for PCI\VEN_8086&DEV_9D31&REV_21
18:40:17:0505 Fu  using 6a546d7e-b4f7-59b3-bf2b-a2c101a13dc7 for PCI\VEN_8086&DEV_9D31
18:40:17:0505 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0505 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:14.2
18:40:17:0505 Fu  performing udev_device_added() on udev
18:40:17:0505 Fu  performing udev_device_added() on nvme
18:40:17:0505 Fu  using 145eb095db7261ae4089be796e4a89f3522a9bd1 for /sys/devices/pci0000:00/0000:00:15.0
18:40:17:0505 Fu  using 93b9ad93-d083-5a0d-a90b-cb6e9c44d15c for PCI\VEN_8086&DEV_9D60&REV_21
18:40:17:0505 Fu  using 96da90c7-642f-5165-bf42-952d531e23f2 for PCI\VEN_8086&DEV_9D60
18:40:17:0505 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0505 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:15.0
18:40:17:0505 Fu  performing udev_device_added() on udev
18:40:17:0505 Fu  performing udev_device_added() on nvme
18:40:17:0505 Fu  using ac6e38a023e4cba1b3d236b2f7eeec16705cd849 for /sys/devices/pci0000:00/0000:00:15.1
18:40:17:0505 Fu  using 539c2ee8-78ac-5ae5-a9d7-3fc1b0b827ae for PCI\VEN_8086&DEV_9D61&REV_21
18:40:17:0505 Fu  using e97e3888-b532-5521-9c20-02934f62f836 for PCI\VEN_8086&DEV_9D61
18:40:17:0505 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0505 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:15.1
18:40:17:0505 Fu  performing udev_device_added() on udev
18:40:17:0505 Fu  performing udev_device_added() on nvme
18:40:17:0505 Fu  using 1d2e8dbf5600ba409dba692d93d70dddd06ad3df for /sys/devices/pci0000:00/0000:00:16.0
18:40:17:0505 Fu  using 286a9248-e36b-5fda-972e-f5660cfc5e94 for PCI\VEN_8086&DEV_9D3A&REV_21
18:40:17:0505 Fu  using 13c76b12-1968-52ec-985a-d03f528c733e for PCI\VEN_8086&DEV_9D3A
18:40:17:0505 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0505 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:16.0
18:40:17:0505 Fu  performing udev_device_added() on udev
18:40:17:0505 Fu  performing udev_device_added() on nvme
18:40:17:0505 Fu  using 8ee39fd33f48f28ab68c09fb046fd0fe1eebcb7b for /sys/devices/pci0000:00/0000:00:17.0
18:40:17:0506 Fu  using 68a0b930-8590-5867-b490-8ba224a7b114 for PCI\VEN_8086&DEV_9D03&REV_21
18:40:17:0506 Fu  using 2b8f660d-b8f9-5390-82d5-4f6dc805cf83 for PCI\VEN_8086&DEV_9D03
18:40:17:0506 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0506 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:17.0
18:40:17:0506 Fu  performing udev_device_added() on udev
18:40:17:0506 Fu  performing udev_device_added() on nvme
18:40:17:0506 Fu  using 210095bfab9420a876a10d174da5de78f73d1d19 for /sys/devices/pci0000:00/0000:00:1c.0
18:40:17:0506 Fu  using bfe6df32-25b8-5c7d-8417-c3056f13ce8a for PCI\VEN_8086&DEV_9D10&REV_F1
18:40:17:0506 Fu  using f8b8f8bc-d872-54f1-8f14-425cdbee4208 for PCI\VEN_8086&DEV_9D10
18:40:17:0506 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0506 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1c.0
18:40:17:0506 Fu  performing udev_device_added() on udev
18:40:17:0506 Fu  performing udev_device_added() on nvme
18:40:17:0506 Fu  using c8d98c4f03163ba91bd8d992de56bc402d63b46e for /sys/devices/pci0000:00/0000:00:1c.0/0000:01:00.0
18:40:17:0506 Fu  using bbdd8ca7-5472-5da5-ac51-e91ab229b17e for PCI\VEN_10EC&DEV_525A&REV_01
18:40:17:0506 Fu  using 275da6fc-d9db-5cde-ac23-ee61f1f40a24 for PCI\VEN_10EC&DEV_525A
18:40:17:0506 Fu  using 90cce275-1f6d-513a-9fd6-ab99de186e5a for PCI\VEN_10EC
18:40:17:0506 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1c.0/0000:01:00.0
18:40:17:0506 Fu  performing udev_device_added() on udev
18:40:17:0506 Fu  performing udev_device_added() on nvme
18:40:17:0506 Fu  using 3af4aace5ad94a02100aafcf7eb64ace336d2743 for /sys/devices/pci0000:00/0000:00:1c.2
18:40:17:0506 Fu  using 1f79cb60-f8ac-58fd-a63d-708455da9598 for PCI\VEN_8086&DEV_9D12&REV_F1
18:40:17:0506 Fu  using abde3196-7c72-53bd-a521-e1266b83bf36 for PCI\VEN_8086&DEV_9D12
18:40:17:0506 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0506 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1c.2
18:40:17:0506 Fu  performing udev_device_added() on udev
18:40:17:0506 Fu  performing udev_device_added() on nvme
18:40:17:0506 Fu  using 6c9a3155a4a85547b1404865c8a8ab37b0829c79 for /sys/devices/pci0000:00/0000:00:1c.2/0000:02:00.0
18:40:17:0506 Fu  using ca4498e4-2f51-56ab-9c24-864e77e77054 for PCI\VEN_168C&DEV_003E&REV_32
18:40:17:0506 Fu  using 9d3a06a4-3e4a-5058-9fd8-55c9c30a0ba5 for PCI\VEN_168C&DEV_003E
18:40:17:0506 Fu  using b95f29b0-44b5-52ab-9963-6eeb24b314e9 for PCI\VEN_168C
18:40:17:0506 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1c.2/0000:02:00.0
18:40:17:0506 Fu  performing udev_device_added() on udev
18:40:17:0506 Fu  performing udev_device_added() on nvme
18:40:17:0507 Fu  using b705332bb14b39bf029db550cb8d5c114e3ec35c for /sys/devices/pci0000:00/0000:00:1c.4
18:40:17:0507 Fu  using da18b740-508f-5b98-8a56-89e3340f23e3 for PCI\VEN_8086&DEV_9D14&REV_F1
18:40:17:0507 Fu  using 85bbf205-665d-5b36-9cd3-ab1524899d3f for PCI\VEN_8086&DEV_9D14
18:40:17:0507 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0507 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1c.4
18:40:17:0507 Fu  performing udev_device_added() on udev
18:40:17:0507 Fu  performing udev_device_added() on nvme
18:40:17:0507 Fu  using 441627a3f3dc6798d15cfafe3b8e90a59593dc3a for /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0
18:40:17:0507 Fu  using e94a0ac3-0aa9-5be0-86d8-d99f38c2a30f for PCI\VEN_8086&DEV_15D3&REV_02
18:40:17:0507 Fu  using 5801f02b-a56b-54dc-a1ff-4853fa318548 for PCI\VEN_8086&DEV_15D3
18:40:17:0507 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0507 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0
18:40:17:0507 Fu  performing udev_device_added() on udev
18:40:17:0507 Fu  performing udev_device_added() on nvme
18:40:17:0507 Fu  using b9deee3463af7fc8089616d8bea2141d8e022dee for /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:00.0
18:40:17:0507 Fu  using e94a0ac3-0aa9-5be0-86d8-d99f38c2a30f for PCI\VEN_8086&DEV_15D3&REV_02
18:40:17:0507 Fu  using 5801f02b-a56b-54dc-a1ff-4853fa318548 for PCI\VEN_8086&DEV_15D3
18:40:17:0507 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0507 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:00.0
18:40:17:0507 Fu  performing udev_device_added() on udev
18:40:17:0507 Fu  performing udev_device_added() on nvme
18:40:17:0507 Fu  using 0158ae7355e6b919784150d3d636ffca78908dc0 for /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:00.0/0000:05:00.0
18:40:17:0507 Fu  using 8a65c3a0-8c78-5745-a688-af823ba04b4f for PCI\VEN_8086&DEV_15D2&REV_02
18:40:17:0507 Fu  using 768e0762-375a-5877-8ac9-38c11adef531 for PCI\VEN_8086&DEV_15D2
18:40:17:0507 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0507 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:00.0/0000:05:00.0
18:40:17:0507 Fu  performing udev_device_added() on udev
18:40:17:0507 Fu  performing udev_device_added() on nvme
18:40:17:0507 Fu  using 4423a0b0f195e59f1a42d4ac334fd094e613af87 for /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:01.0
18:40:17:0507 Fu  using e94a0ac3-0aa9-5be0-86d8-d99f38c2a30f for PCI\VEN_8086&DEV_15D3&REV_02
18:40:17:0507 Fu  using 5801f02b-a56b-54dc-a1ff-4853fa318548 for PCI\VEN_8086&DEV_15D3
18:40:17:0507 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0507 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:01.0
18:40:17:0507 Fu  performing udev_device_added() on udev
18:40:17:0507 Fu  performing udev_device_added() on nvme
18:40:17:0507 Fu  using a7a0d6eaac3b26b1b6a5ec4710c27190273feddb for /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:02.0
18:40:17:0508 Fu  using e94a0ac3-0aa9-5be0-86d8-d99f38c2a30f for PCI\VEN_8086&DEV_15D3&REV_02
18:40:17:0508 Fu  using 5801f02b-a56b-54dc-a1ff-4853fa318548 for PCI\VEN_8086&DEV_15D3
18:40:17:0508 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0508 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:02.0
18:40:17:0508 Fu  performing udev_device_added() on udev
18:40:17:0508 Fu  performing udev_device_added() on nvme
18:40:17:0508 Fu  using b5b99f90b351edd879b32c1572d45b1d30df6646 for /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:02.0/0000:39:00.0
18:40:17:0508 Fu  using 7d3fed74-877a-5a9a-9bd6-74f31065a9df for PCI\VEN_8086&DEV_15D4&REV_02
18:40:17:0508 Fu  using 1bc7322b-f7ef-5594-9a1e-a40c74573b18 for PCI\VEN_8086&DEV_15D4
18:40:17:0508 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0508 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:02.0/0000:39:00.0
18:40:17:0508 Fu  performing udev_device_added() on udev
18:40:17:0508 Fu  performing udev_device_added() on nvme
18:40:17:0508 Fu  using a81f32b211a4d5a9b14d38c4a17d36ebc2a52f32 for /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:04.0
18:40:17:0508 Fu  using e94a0ac3-0aa9-5be0-86d8-d99f38c2a30f for PCI\VEN_8086&DEV_15D3&REV_02
18:40:17:0508 Fu  using 5801f02b-a56b-54dc-a1ff-4853fa318548 for PCI\VEN_8086&DEV_15D3
18:40:17:0508 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0508 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:04.0
18:40:17:0508 Fu  performing udev_device_added() on udev
18:40:17:0508 Fu  performing udev_device_added() on nvme
18:40:17:0508 Fu  using c15f4b25f89de3cb42e31598f01e44d1d0047416 for /sys/devices/pci0000:00/0000:00:1f.0
18:40:17:0508 Fu  using b9ae973a-79ec-55ca-a038-2ccfcb5b94c6 for PCI\VEN_8086&DEV_9D4E&REV_21
18:40:17:0508 Fu  using e7f0e2f6-3fe9-5cca-b24e-6ae4026f0974 for PCI\VEN_8086&DEV_9D4E
18:40:17:0508 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0508 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1f.0
18:40:17:0508 Fu  performing udev_device_added() on udev
18:40:17:0508 Fu  performing udev_device_added() on nvme
18:40:17:0508 Fu  using e359acb6fdfa4603d5143e4ce7f420c28536fd4a for /sys/devices/pci0000:00/0000:00:1f.2
18:40:17:0508 Fu  using bdc79851-62c9-5c44-8c1e-5a58d9da1af4 for PCI\VEN_8086&DEV_9D21&REV_21
18:40:17:0508 Fu  using e3290f75-28b6-58c2-962a-d36145469508 for PCI\VEN_8086&DEV_9D21
18:40:17:0508 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0508 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1f.2
18:40:17:0508 Fu  performing udev_device_added() on udev
18:40:17:0508 Fu  performing udev_device_added() on nvme
18:40:17:0508 Fu  using c187b91629d45be681e19682f9bfe0d2eb8d2e63 for /sys/devices/pci0000:00/0000:00:1f.3
18:40:17:0509 Fu  using 8429601d-fa69-5c91-a068-f9215e436f37 for PCI\VEN_8086&DEV_9D71&REV_21
18:40:17:0509 Fu  using d8e3bccb-56b2-5e10-a424-8b81d92a6c43 for PCI\VEN_8086&DEV_9D71
18:40:17:0509 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0509 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1f.3
18:40:17:0509 Fu  performing udev_device_added() on udev
18:40:17:0509 Fu  performing udev_device_added() on nvme
18:40:17:0509 Fu  using 04d519c5e2a7830c11b035f672f56185766b8636 for /sys/devices/pci0000:00/0000:00:1f.4
18:40:17:0509 Fu  using d7bef7ad-3f53-5c4f-9ed2-efb361ccd6c4 for PCI\VEN_8086&DEV_9D23&REV_21
18:40:17:0509 Fu  using 7e38d50b-bf42-5a6b-b32e-8c675104a819 for PCI\VEN_8086&DEV_9D23
18:40:17:0509 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0509 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1f.4
18:40:17:0509 Fu  performing udev_device_added() on udev
18:40:17:0509 Fu  performing udev_device_added() on nvme
18:40:17:0509 Fu  0 devices with subsystem nvme
18:40:17:0509 Fu  FuHistory: trying to open database '/var/lib/fwupd/pending.db'
18:40:17:0528 Fu  FuHistory: got schema version of 2
18:40:17:0528 Fu  FuHistory: get devices
18:40:17:0528 Fu  FuHistory: got sql result 1f3ddf782a928951c61a32bdb0bd140da3f33ac7
18:40:17:0528 Fu  Emitting PropertyChanged('Status'='idle')
18:40:17:0528 Fu  Emitting PropertyChanged('Status'='idle')
18:40:17:0530 Fu  Daemon ready for requests
18:40:17:0530 ThunderboltPower uevent for domain0: (/sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:00.0/0000:05:00.0/domain0) add
18:40:17:0530 ThunderboltPower Setting timeout to 20 seconds
18:40:17:0530 Thunderbolt uevent for /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:00.0/0000:05:00.0/domain0: add
18:40:17:0531 Fu  using 0158ae7355e6b919784150d3d636ffca78908dc0 for /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:00.0/0000:05:00.0
18:40:17:0531 Fu  using 8a65c3a0-8c78-5745-a688-af823ba04b4f for PCI\VEN_8086&DEV_15D2&REV_02
18:40:17:0531 Fu  using 768e0762-375a-5877-8ac9-38c11adef531 for PCI\VEN_8086&DEV_15D2
18:40:17:0531 Fu  using afb8c4ec-b9a5-5cd3-aaef-cc976ce37eb8 for PCI\VEN_8086
18:40:17:0531 Fu  no plugin specified for udev device /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:00.0/0000:05:00.0
18:40:17:0531 Fu  performing udev_device_added() on udev
18:40:17:0531 Fu  performing udev_device_added() on nvme
18:40:17:0536 ThunderboltPower uevent for 0-0: (/sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:00.0/0000:05:00.0/domain0/0-0) add
18:40:17:0536 ThunderboltPower Setting timeout to 20 seconds
18:40:17:0536 Thunderbolt uevent for /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:00.0/0000:05:00.0/domain0/0-0: add
18:40:17:0537 Thunderbolt adding udev device: c7010000-0070-6f18-23fd-a048d7c0301d at /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:00.0/0000:05:00.0/domain0/0-0
18:40:22:0011 Fu  using d10c5f0ed12c6dc773f596b8ac51f8ace4355380 for c7010000-0070-6f18-23fd-a048d7c0301d
18:40:22:0011 Fu  using 4eeb9d07-a96c-56d6-92d3-4a23ee7a6e4a for TBT-00d407e6
18:40:22:0011 Fu  emit added from thunderbolt: d10c5f0ed12c6dc773f596b8ac51f8ace4355380
18:40:22:0011 Fu  performing fu_plugin_device_registered() on dell
18:40:22:0011 Fu  performing fu_plugin_device_registered() on uefi
18:40:22:0011 Fu  performing fu_plugin_device_registered() on thunderbolt_power
18:40:22:0015 ThunderboltPower Bolt force power support: 1
18:40:22:0015 Fu  ::added d10c5f0ed12c6dc773f596b8ac51f8ace4355380
18:40:22:0015 Fu  performing a recoldplug
18:40:22:0016 Fu  performing coldplug_prepare() on dell
18:40:22:0547 Dell Toggled dock mode to 1
18:40:22:0547 Fu  got coldplug delay of 2000ms, global maximum is now 2000ms
18:40:22:0547 Fu  sleeping for 2000ms
18:40:24:0547 As  run FuEngine:coldplug
18:40:24:0547 As  run FuEngine:coldplug{amt}
18:40:24:0547 As  run FuEngine:coldplug{udev}
18:40:24:0548 As  run FuEngine:coldplug{dell_esrt}
18:40:24:0548 As  run FuEngine:coldplug{ebitdo}
18:40:24:0548 As  run FuEngine:coldplug{steelseries}
18:40:24:0548 As  run FuEngine:coldplug{dfu}
18:40:24:0548 As  run FuEngine:coldplug{nitrokey}
18:40:24:0548 As  run FuEngine:coldplug{csr}
18:40:24:0548 As  run FuEngine:coldplug{unifying}
18:40:24:0548 As  run FuEngine:coldplug{superio}
18:40:24:0548 As  run FuEngine:coldplug{wacomhid}
18:40:24:0548 As  run FuEngine:coldplug{altos}
18:40:24:0549 As  run FuEngine:coldplug{upower}
18:40:24:0549 As  run FuEngine:coldplug{colorhug}
18:40:24:0549 As  run FuEngine:coldplug{thunderbolt}
18:40:24:0549 Fu  performing recoldplug() on thunderbolt
18:40:24:0550 Thunderbolt adding udev device: c7010000-0070-6f18-23fd-a048d7c0301d at /sys/devices/pci0000:00/0000:00:1c.4/0000:03:00.0/0000:04:00.0/0000:05:00.0/domain0/0-0
18:40:24:0550 Thunderbolt ignoring duplicate tbt__sys_devices_pci0000_00_0000_00_1c_4_0000_03_00_0_0000_04_00_0_0000_05_00_0_domain0_0_0
18:40:24:0550 As  run FuEngine:coldplug{dell}
18:40:24:0551 Fu  performing recoldplug() on dell
18:40:24:0555 Dell TPM HW version: 0x0
18:40:24:0555 Dell TPM Status: 0x207
18:40:24:0555 Dell Creating primary TPM GUID 07e6-2.0 and secondary TPM GUID 07e6-1.2
18:40:24:0555 Fu  using 370e10407b1f04ade798a9f1d3e1fa57c67750c3 for DELL-cb8da68d-cd80-5f5b-8fef-038383adbb83lu
18:40:24:0555 Dell XPS 13 9370 TPM 2.0 updating disabled due to TPM ownership
18:40:24:0555 Fu  emit device-register from dell: 370e10407b1f04ade798a9f1d3e1fa57c67750c3
18:40:24:0555 Fu  performing fu_plugin_device_registered() on dell
18:40:24:0555 Fu  performing fu_plugin_device_registered() on uefi
18:40:24:0555 Fu  emit added from uefi: 370e10407b1f04ade798a9f1d3e1fa57c67750c3
18:40:24:0555 Fu  performing fu_plugin_device_registered() on dell
18:40:24:0555 Fu  performing fu_plugin_device_registered() on thunderbolt_power
18:40:24:0555 Fu  ignoring device 370e10407b1f04ade798a9f1d3e1fa57c67750c3 [uefi] existing device 088df415cdee883ec89563e41e6d495924250174 [amt] already exists
18:40:24:0555 Fu  performing fu_plugin_device_registered() on thunderbolt_power
18:40:24:0555 Dell System 07e6 does not offer TPM modeswitching
18:40:24:0555 As  run FuEngine:coldplug{flashrom}
18:40:24:0555 As  run FuEngine:coldplug{redfish}
18:40:24:0555 As  run FuEngine:coldplug{nvme}
18:40:24:0555 As  run FuEngine:coldplug{uefi}
18:40:24:0555 As  run FuEngine:coldplug{thunderbolt_power}
18:40:24:0555 Fu  performing recoldplug() on thunderbolt_power
18:40:24:0556 ThunderboltPower Bolt force power support: 1
18:40:24:0556 As  run FuEngine:coldplug{synapticsmst}
18:40:24:0556 Fu  performing recoldplug() on synapticsmst
18:40:24:0560 SynapticsMST Skipping previously added device MST-DIRECT-drm_dp_aux1-0-0
18:40:24:0590 SynapticsMST No cascade device found: remote command failed: 3
18:40:24:0614 SynapticsMST No cascade device found: remote command failed: 3
18:40:24:0885 SynapticsMST No device found on drm_dp_aux2: failed to read device: failed to read
18:40:24:0885 SynapticsMST No device found on drm_dp_aux0: no device
18:40:24:0885 Fu  performing coldplug_cleanup() on dell
18:40:25:0398 Dell Toggled dock mode to 0
18:40:25:0398 Fu  using plugins: amt, udev, ebitdo, steelseries, dfu, nitrokey, csr, unifying, superio, wacomhid, altos, upower, colorhug, thunderbolt, dell, flashrom, nvme, uefi, thunderbolt_power, synapticsmst
18:40:25:0398 Fu  emit added from udev: ca9f93cb97772027f45ba73c86cf4980fb02745b
18:40:25:0398 Fu  performing fu_plugin_device_registered() on dell
18:40:25:0399 Fu  performing fu_plugin_device_registered() on uefi
18:40:25:0399 Fu  performing fu_plugin_device_registered() on thunderbolt_power
18:40:25:0399 Fu  ::added ca9f93cb97772027f45ba73c86cf4980fb02745b
18:40:25:0399 Fu  registering D-Bus service
################################                                                       @4798ms FuEngine:load 4798ms
#                                                                                      @0002ms AsStore:load-installed{/usr/share/fwupd/metainfo} 1ms
#                                                                                      @0011ms FuEngine:load-quirks 1ms
#                                                                                      @0022ms AsStore:store-from-file{/var/lib/fwupd/remotes.d/dell-embargo/metadata.xml.gz} 10ms
#                                                                                      @0022ms AsStore:store-from-root 2ms
#                                                                                      @0037ms AsStore:store-from-file{/var/lib/fwupd/remotes.d/lvfs/metadata.xml.gz} 12ms
#                                                                                      @0037ms AsStore:store-from-root 3ms
#                                                                                      @0063ms FuEngine:load-plugins 10ms
#                                                                                      @0066ms FuEngine:setup 2ms
#                                                                                      @0065ms FuEngine:setup{upower} 2ms
                 ########                                                              @3794ms FuEngine:coldplug 1195ms
                 #                                                                     @2658ms FuEngine:coldplug{amt} 59ms
                  #                                                                    @2678ms FuEngine:coldplug{unifying} 18ms
                  #                                                                    @2690ms FuEngine:coldplug{dell} 11ms
                  #                                                                    @2767ms FuEngine:coldplug{uefi} 76ms
                  #                                                                    @2804ms FuEngine:coldplug{thunderbolt_power} 36ms
                   ###                                                                 @3277ms FuEngine:coldplug{synapticsmst} 473ms
                                                                                #####  @12668ms FuEngine:coldplug 851ms
                                                                                #      @11820ms FuEngine:coldplug{thunderbolt} 1ms
                                                                                #      @11824ms FuEngine:coldplug{dell} 4ms
                                                                                #      @11825ms FuEngine:coldplug{thunderbolt_power} 1ms
                                                                                ##     @12154ms FuEngine:coldplug{synapticsmst} 328ms
18:40:25:0405 Fu  FuMain: acquired name: org.freedesktop.fwupd
18:40:37:0025 Unifying device connection event, do something
18:40:37:0544 ThunderboltPower Bolt force power support: 1
18:40:37:0545 ThunderboltPower Setting force power to 0 using bolt
```

superm1 commented:
> This sounds like a bug; if the platform ID is NULL we're never supposed to compare it... Got any more info?

Fixed the bug in the above commit for `fu-device-list.c`.
